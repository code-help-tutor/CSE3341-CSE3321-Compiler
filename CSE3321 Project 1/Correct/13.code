precedure int x, y, z; A (a, b, c) begin a = b < c; begin in x, y, z; A (x, y, z); end

