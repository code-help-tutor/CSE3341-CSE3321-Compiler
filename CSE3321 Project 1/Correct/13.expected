ID[precedure]
ID[int]
ID[x]
COMMA
ID[y]
COMMA
ID[z]
SEMICOLON
ID[A]
LPAREN
ID[a]
COMMA
ID[b]
COMMA
ID[c]
RPAREN
BEGIN
ID[a]
EQUAL
ID[b]
LESS
ID[c]
SEMICOLON
BEGIN
IN
ID[x]
COMMA
ID[y]
COMMA
ID[z]
SEMICOLON
ID[A]
LPAREN
ID[x]
COMMA
ID[y]
COMMA
ID[z]
RPAREN
SEMICOLON
END
